src/test/java/com/mael/ttt/GameTest.java

```diff
@@ -57,7 +62,7 @@ public void runEndsTheGameIfFull() {
     public void checkForFullPrintsBoardAndMsgs() {
         spy.setInputs("1", "1", "2", "3", "7", "4", "6", "5", "9", "8", "n");
         game.run();
-        assertThat(spy.printedMessage(), containsString("X O X \nX X O \nO X O \n" + UserInterface.ISFULL + UserInterface.GAMEOVER));
+        assertThat(spy.printedMessage(), containsString(playerMark + " " + opponentMark + " " + playerMark + " \n" + playerMark + " " + playerMark + " " + opponentMark + " \n" + opponentMark + " " + playerMark + " " + opponentMark + " \n" + UserInterface.ISFULL + UserInterface.GAMEOVER));
```

Comment:
I did too!
I changed it in case I wanted to use octopuses or smileys or H and R etc. in the future...
Gonna think in a better way of doing this :+1: